vagrant-default.yml

```diff
@@ -64,6 +64,7 @@ role:
     postgresql: false
     redis: false
     blackfire: false
+    rabbitmq: false
```

Comment:
@scuben please provide add a rabbitmq with user / password and the config as yaml which can be coverted by a jinja template https://www.rabbitmq.com/configure.html#config-file-formats